cookbook/integrations/flytekit_plugins/duckdb_examples/example.py

```diff
@@ -0,0 +1,157 @@
+"""
+DuckDB Example
+--------------
+
+In this example, we will see how to run SQL queries using DuckDB. 
+
+First, import the necessary libraries.
+"""
+
+import json
+from typing import List
+
+import pandas as pd
+import pyarrow as pa
+from flytekit import kwtypes, task, workflow
+from flytekit.types.structured.structured_dataset import StructuredDataset
+from flytekitplugins.duckdb import DuckDBQuery
+from typing_extensions import Annotated
+
+# %%
+# A simple ``SELECT`` query
+# =========================
+#
+# To run a simple ``SELECT`` query, initialize a ``DuckDBQuery`` task.
+simple_duckdb_query = DuckDBQuery(
+    name="simple_task",
+    query="SELECT SUM(a) FROM mydf",
+    inputs=kwtypes(mydf=pd.DataFrame),
+)
+
+
+# %%
+# You can invoke the task from within a :py:func:`~flytekit:flytekit.workflow` and return both a Pandas dataframe and a PyArrow table.
+# The query will be executed on a Pandas dataframe, and the resulting output can belong to any StructuredDataset-compatible type.
+@task
+def get_pandas_df() -> pd.DataFrame:
+    return pd.DataFrame({"a": [1, 2, 3]})
+
+
+@workflow
+def pandas_wf() -> pd.DataFrame:
+    return simple_duckdb_query(mydf=get_pandas_df())
+
+
+@workflow
+def arrow_wf() -> pa.Table:
+    return simple_duckdb_query(mydf=get_pandas_df())
+
+
+if __name__ == "__main__":
+    print(f"Running pandas_wf()... {pandas_wf()}")
+    print(f"Running arrow_wf()... {arrow_wf()}")
+
+# %%
+# SQL query on Parquet file
+# =========================
+#
+# DuckDB enables direct querying of a parquet file without the need for intermediate conversions to a database.
+# If you wish to execute a SQL query on a parquet file stored in a public S3 bucket, you can use the ``httpfs`` library by installing and loading it.
+# Simply send the parquet file as a parameter to the ``SELECT`` query.
+parquet_duckdb_query = DuckDBQuery(
+    name="parquet_query",
+    query=[
+        "INSTALL httpfs",
+        "LOAD httpfs",
+        """SELECT hour(lpep_pickup_datetime) AS hour, count(*) AS count FROM READ_PARQUET(?) GROUP BY hour""",
+    ],
+    inputs=kwtypes(params=List[str]),
+)
+
```

Comment:
can we move the last two lines 
```
# It is important to note that multiple commands can be executed within a single ``DuckDBQuery``.
# However, it is essential to ensure that the last command within the query is a "SELECT" query to retrieve data in the end.
```

up to this section so that users know that `query` can be a list of strings for issuing multiple commands?